BlogTemplate/wwwroot/css/colors.css

```diff
@@ -5,7 +5,7 @@
     --hover-color: #7f7f7f;
```

Comment:
If we're not using colors.css anymore, we should remove the file.  Otherwise it's clutter that we'll need to clean up later.